examples/julia/make.jl

```diff
@@ -0,0 +1,7 @@
+using Remark, FileWatching
+
+while true
+    Remark.slideshow(@__DIR__; options = Dict("ratio" => "16:9"))
```

Comment:
maybe include the `title` setting in this example since i imagine it is going to be nearly universally used